test/routes/user/email.test.ts

```diff
@@ -109,4 +109,79 @@ describe('user email', () => {
       .send({ email: newEmail, password })
       .expect(200);
   });
+
+  it('change email with redirect', async () => {
+    await request.post('/change-env').send({
+      AUTH_DISABLE_NEW_USERS: false,
+      AUTH_EMAIL_SIGNIN_EMAIL_VERIFIED_REQUIRED: false,
+    });
+
+    let accessToken = '';
+
+    const email = 'asdasd@asdasd.com';
+    const password = '123123123';
+    const options = {
+      redirectTo: 'http://localhost:3000/email-redirect'
+    }
+
+    await request
+      .post('/signup/email-password')
+      .send({ email, password })
+      .expect(200);
+
+    const { body }: { body: SignInResponse } = await request
+      .post('/signin/email-password')
+      .send({ email, password })
+      .expect(200);
+
+    expect(body.session).toBeTruthy();
+
+    if (!body.session) {
+      throw new Error('session is not set');
+    }
+
+    accessToken = body.session.accessToken;
```

Comment:
@alveshelio is there a reason to declare `accessToken` in line #119 if you are only using it here? 

`const accessToken = body.session.accessToken;`

Reply:
No particular reason :P, sorry about that, I should have spent a little more time understanding how you guys are testing and specially how everything works :)
I've simply copied a test and adjusted for my use case.

Reply:
I went ahead and refactored a bit the tests and moved some logic to avoid duplication and actually I've added tests for email verification. Let me know if this is good.